.github/workflows/build-nightly-release.yml

```diff
@@ -0,0 +1,14 @@
+name: Build Nightly Release
+on:
+  schedule:
+    # Run every day at 2:00 AM
+    - cron: "0 2 * * *"
+
+jobs:
+  build-nightly-release:
+    if: github.repository == 'icerpc/icerpc-csharp'
+    uses: ./.github/workflows/build-release.yml
+    secrets: inherit
+    with:
+      channel: nightly
+      publish: true
```

Comment:
This schedule nightly release 2:00 AM every day, published to ZeroC NuGet nightly repository.